events/static/js/app/NewEventForm.js

```diff
@@ -0,0 +1,245 @@
+import React from 'react'
+import axios from 'axios'
+import FloatingActionButton from 'material-ui/FloatingActionButton';
+import ContentAdd from 'material-ui/svg-icons/content/add';
+import DatePicker from 'material-ui/DatePicker';
+import TimePicker from 'material-ui/TimePicker';
+import TextField from 'material-ui/TextField';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import {Container, Row, Col} from 'react-grid-system';
+import Map from './map';
+
+
+
+
+
+class Form extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = (
+      {
+        title: '',
+        start_date: this.initialDate().start_date / 1000,
```

Comment:
You mix variable names convention. You use CamelCase and underscores together. Ideally, JS code should always have CamelCase.